cmd/kyma/alpha/disable/module/module.go

```diff
@@ -157,7 +157,7 @@ func disableModule(modules []v1beta1.Module, name, channel string) ([]v1beta1.Mo
 			continue
 		}
 		if channel != "" {
-			if mod.Channel != channel {
+			if mod.Channel != "" && mod.Channel != channel {
```

Comment:
1. It's better to have a unit test for this disableModule function.
2.Just want to point out, the validation of the given `channel` is a valid `kyma.spec.channel` is missing, however, it's being implemented in this [PR](https://github.com/kyma-project/cli/pull/1583/files#diff-8c9ab491077df3621b75d4945099d239b8623f5d657388b8b698781ac05cc5d5R138-R143)